src/hooks/useRelatedVideos.js

```diff
@@ -1,35 +1,35 @@
-import { useState, useEffect, useCallback } from 'react';
-import youtube from '../apis/youtube';
-
-const useRelatedVideos = (query) => {
-  const [isLoading, setIsLoading] = useState(false);
-  const [videos, setVideos] = useState([]);
-
-  const handleSubmit = useCallback(async (query) => {
-    setIsLoading(true);
-
-    try {
-      const response = await youtube.get('/search', {
-        params: {
-          relatedToVideoId: query,
-          type: 'video',
-          part: 'snippet',
-        },
-      });
-
-      setVideos(response.data.items);
-      setIsLoading(false);
-    } catch (error) {
-      console.log(error);
-      setIsLoading(false);
-    }
-  }, []);
-
-  useEffect(() => {
-    handleSubmit(query);
-  }, [query, handleSubmit]);
-
-  return [isLoading, videos];
-};
-
-export default useRelatedVideos;
+import { useState, useEffect, useCallback } from 'react';
+import youtube from '../apis/youtube';
+
+const useRelatedVideos = (query) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [videos, setVideos] = useState([]);
+
+  const handleSubmit = useCallback(async (search) => {
+    setIsLoading(true);
+
+    try {
+      const response = await youtube.get('/search', {
+        params: {
+          relatedToVideoId: search,
+          type: 'video',
+          part: 'snippet',
+        },
+      });
+
+      setVideos(response.data.items);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+      setIsLoading(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    handleSubmit(query);
+  }, [query, handleSubmit]);
+
+  return [isLoading, videos];

```

Comment:
It is better to return an object like:
```suggestion
  return  { isLoading, videos };

```

So that you can destructure when using this hook.